app/models/form_submission.rb

```diff
@@ -0,0 +1,27 @@
+# == Schema Information
+#
+# Table name: form_submissions
+#
+#  id              :bigint           not null, primary key
+#  created_at      :datetime         not null
+#  updated_at      :datetime         not null
+#  organization_id :bigint           not null
+#  person_id       :bigint           not null
+#
+# Indexes
+#
+#  index_form_submissions_on_organization_id  (organization_id)
+#  index_form_submissions_on_person_id        (person_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (organization_id => organizations.id)
+#  fk_rails_...  (person_id => people.id)
+#
+class FormSubmission < ApplicationRecord
```

Comment:
I am also a fan of documenting in the app where we can. Can we add a line or two of comments above the class definition to explain what this class does? I think it's quite an important piece of the architecture managing adopter data. 

Something like...
```
The FormSubmission model binds together a set of submitted answers from an adopter into a single submission. This represents a snapshot in time. If an adopter submits answers either through an in-app form or a third party form that is uploaded to the app, we will have a set of answers belonging to a form submission. This can be valid for a period of time to allow the user to apply to adopt multiple pets. If this time expires, or the person's contet changes and they need to update the form, we now have a new set of answers and a new form submission.
```
```

Reply:
@Gabe-Torres can you add the above comments above the class definition?